api-description/forms/pk/post.json

```diff
@@ -2,22 +2,36 @@
     "description": "Endpoint to answer a form",
     "url": "http://localhost:8000/forms/1",
     "fields": {
-        "answers": {
-            "1": "Friends",
-            "2": "one_time",
-            "3": ["two_times", "three_times"],
-            "4": 5
-        }
+        "survey": 1,
```

Comment:
Yeah, Kevin said that yesterday.  I'm not 100% sure on how that works programmatically, I might have to 'force it' in the view.